core/src/main/java/com/google/instrumentation/stats/MutableView.java

```diff
@@ -19,8 +19,10 @@
 import com.google.instrumentation.stats.ViewDescriptor.DistributionViewDescriptor;
 import com.google.instrumentation.stats.ViewDescriptor.IntervalViewDescriptor;
 import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
I was wondering whether we should sort the Aggregations in a View.  @a-veitch thought that we shouldn't guarantee an order, so that we have the flexibility to change the implementation later.  If we don't guarantee the order, then I don't think we should test the order of the Aggregations.

I think we could deal with the unknown order by adding a class with some testing utilities, e.g., a method that takes a View and returns the Aggregations in sorted order.